scripts/container.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# move into base directory, in case this script is not executed via `make container`
+cd "$(dirname "$0")/.."
+
+# normalize branch name to reflect a valid image name
+BRANCH=$(git branch --show-current | sed 's/[^a-z0-9-]/_/ig')
+TAG=gluon:${BRANCH}
+
+if [ "$(command -v podman)" ]
+then
+	(
+		cd contrib/docker
+		podman build -t "${TAG}" .
+	)
```

Comment:
```suggestion
	podman build -t "${TAG}" contrib/docker
```

Reply:
Ah, I somewhat expected it to still look at the cwd for build context. Great if that is sufficient.